plasmapy/analysis/time_series/running_moments.py

```diff
@@ -0,0 +1,128 @@
+"""Functionality to calculate running moments of time series"""
+import numpy as np
+
+
+def running_mean(signal, radius):
+    """
+    Calculate the running mean of a sequence
+
+    Parameters
+    ----------
+    signal: |array_like|
+        signal to be averaged.
+
+    radius: int
+        window size is 2*radius+1
+
+    Returns
+    -------
+    |array_like|
+        running mean of signal with length len(signal) - 2*radius
+
+    Raises
+    ------
+    'ValueError'
+        if len(signal <= 2 * radius)
+
+    Example
+    -------
+    >>> from plasmapy.analysis.time_series.running_moments import running_mean
+    >>> running_mean([1,2,3,4], 1)
+    array([2., 3.])
+    """
+    if len(signal) <= 2 * radius:
+        raise ValueError("len(signal) must be bigger than 2*radius")
+
+    window = 2 * radius + 1
+    run_mean = np.cumsum(signal, dtype=float)
+    run_mean[window:] = run_mean[window:] - run_mean[:-window]
+    return run_mean[window - 1 :] / window
+
+
+def running_moment(signal, radius, moment=1, time=None):
+    """
+    Calculate either the running mean, standard deviation, skewness or excess kurtosis of a sequence
+
+    Parameters
+    ----------
+    signal: |array_like|
+       signal to be averaged
+
+    radius: int
+        window size is 2*radius +1 for running mean and 4*radius + 1 for higher moments
+
+    moment: int
+        which running moment to compute
+                1: running mean.
+                2: running standard deviation.
+                3: running skewness
+                4: running excess kurtosis
```

Comment:
After adding the `api_static` pages, it would be worth checking the Read the Docs documentation preview to make sure this is showing up the way that you intended in the online documentation.

Reply:
It looks like the formatting in the [doc preview](https://plasmapy--1803.org.readthedocs.build/en/1803/api/plasmapy.analysis.time_series.running_moments.running_moment.html#running-moment) is coming out a little weird.  I can never remember the formatting for lists in parameter descriptions, but there are a few examples in [`thomson.py`](https://github.com/PlasmaPy/PlasmaPy/blob/17aecc2aa9dcfbaa03252230f51ab0e229cb7759/plasmapy/diagnostics/thomson.py#L688) for unordered lists.